packages/vite/package.json

```diff
@@ -110,7 +110,7 @@
     "postcss-import": "^15.0.0",
     "postcss-load-config": "^4.0.1",
     "postcss-modules": "^5.0.0",
-    "resolve.exports": "^1.1.0",
+    "resolve.exports": "npm:@alloc/resolve.exports@^1.1.0",
```

Comment:
we don't need to change this to use a fork. the current `resolve.exports` version used by Vite supports subpath imports